examples/src/js/BasicExampleObject.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import CheckboxTree from 'react-checkbox-tree';
+
+const initialNodes = {
+    label: 'Root',
```

Comment:
This is something I put in based upon my use and something I read in one of the issues somewhere. I will state my reasoning here and you can decide the path forward as this is your component.

In my use I may have several JSON configuration files which are used for input to the CheckTree. Each has a root node with a description of that tree. The User logs in and can choose from the configurations available to them. My current proposal allows passing this root node object in as the `nodes` prop.  In this case, the root node is the same type object as as each child node object. I can achieve the same result by passing the root node's `children` array as you originally set it up.

I have also allowed the `nodes` prop to be passed in as an array. It is then put into a generated root object so that the processing could be handled the same as when a root object is passed in as described above.

One advantage of having the root object is that you can then add a prop, `showRoot`, to allow having a single node at the top of the tree. This allows you to collapse the tree to one line.

By defaulting `showRoot` to `false` you will have the same result as you originally provided. I have not yet added this prop but had intended to. 

The `showRoot` prop will fix some of the concerns  in issue #143. With `showRoot === true` and 'noCascade === false` you do not need a 'checkAll` or 'unCheckAll`.  Just check or uncheck the root.  

That is my case for adding the capability to input a root node.  I have kept the ability to input an array of nodes for compatibility with previous versions. I can set it up with only the root node option, only the array of nodes option or both.  You choose.